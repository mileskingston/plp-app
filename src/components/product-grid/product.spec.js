import React from 'react';
import renderer from 'react-test-renderer';
import Product from './product';

describe('Product Component', () => {
  const product = {
    name: 'Item name',
    thumbnail: '/image/url',
    price: [
      {
        value: '260.0'
      }
    ]
  };

  it('renders component', () => {
    let component = renderer.create(<Product product={product} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});