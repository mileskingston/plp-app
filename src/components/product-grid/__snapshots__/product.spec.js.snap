// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Component renders component 1`] = `
<div
  className="product"
>
  <a
    href="/"
  >
    <img
      alt="Item name"
      className="product__image"
      src="https://www.demo.salmon-gcp.com/image/url"
    />
    <h2
      className="product__name"
    >
      Item name
    </h2>
    <p
      className="product__price"
    >
      $260.0
    </p>
  </a>
</div>
`;
